students/WWhite/lesson04/trigrams.py

```diff
@@ -0,0 +1,48 @@
+import string, random
+
+file = open('sherlock_small.txt', 'r')
+file_text = file.read().lower()
+file.close()
+
+translator = str.maketrans(string.punctuation, ' '*len(string.punctuation))
+
+new_file_text = file_text.translate(translator).split()
+
+
+def trigrams(list):
+    trigram_list = []
+    for i in range(len(list)-2):
+        trigram_list.append(('{} {}'.format(list[i], list[i+1]), list[i+2]))
+    return trigram_list
```

Comment:
Seems like you could build the dict directly while looping over the text, rather than use an intermediary list